root/etc/cont-init.d/69-geoip2influx

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/with-contenv bash
+
+if [ ! -d /usr/lib/python3.8/site-packages/influxdb ]; then
+  echo '-------------------------------------------------------------'
+  echo '| Mod by Gilbn                                              |'
+  echo '| Running installation of required modules for Geoip2Influx |'
+  echo '-------------------------------------------------------------'
+  pip3 install --no-cache-dir -U \
+    geoip2==3.0.0 \
+    geohash2==1.1 \
+    influxdb==5.3.0 \
+    IPy==1.0
+fi
+
+if [ ! -d /config/geoip2db ]; then
+    echo '-------------------------------------'
+    echo '| Creating /config/geoip2db/ folder |'
+    echo '-------------------------------------'
+    mkdir  /config/geoip2db
+fi
+
+# Fetch latest or update existing
+if [[ ! -d geoip2influx/.git ]]; then
+  echo '---------------------------'
+  echo '| Installing Geoip2Influx |'
+  echo '---------------------------'
+  git clone --progress --verbose https://github.com/gilbN/geoip2influx.git
```

Comment:
Does this have to reside in the config folder? Wouldn't it be better to ship this inside the mod image (tarball)? We can set a jenkins trigger to build and push the mod whenever there is an upstream release or commit

Reply:
Sounds like a good idea!

Reply:
The python script creates the `geoip2influx.log` file in the same directory as itself, so we need to touch and symlink that to /log/geoip2influx/ for example.